### ISSUE TITLE
Aktualizacja stocku

### ISSUE THREAD

**Issue by dontbesoshy** [allegro/allegro-api#9309]
Dzień dobry,

jeżeli ktoś kupi ofertę na allegro to czy poprzez API mogę wysłać request do mojej aplikacji aby obniżyć stock magazynowy? 
Z reguły muszę to zawsze robić ręcznie

**Comment by AureliuszBrussy** [comment]
Możesz korzystać z dziennika ofertowego ([GET /sale/offer-events](https://developer.allegro.pl/documentation/#operation/getOfferSmartClassificationGET)) aby sprawdzać zmiany liczby sztuk w ofercie (np. w wyniku sprzedaży) i używać te dane do aktualizacji stocku w swojej aplikacji.

**Comment by dontbesoshy** [comment]
czy to nie zaburzy spójności? mam u sibeie stok 10, na allegro stock 10, na allegro zostanie zakupiona oferta z (1sztuka) allegro ma stock 9, u siebie mam 10, ktoś w tym czasie kupi u mnie 2szt i mamy zgrzyt 9 vs 8 sztuk. Bo zakładam że to musiałbym wysyłać jakimś cronem dla "OFFER_STOCK_CHANGED"

**Comment by AureliuszBrussy** [comment]
Tak, w tym przypadku wchodzi w grę cron, nie udostępniamy webhooków. Jeżeli chcesz uzyskać spójność pomiędzy swoim stockiem (który może być zmieniony przez zakup np. z innego źródła niż Allegro) - musiałbyś wysyłać także zmianę stocka do Allegro. Źródłem prawdy o stocku musiałby być Twój system.
